org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/files/generateAdaptorApplication.mtl

```diff
@@ -102,11 +106,7 @@ import [javaClassFullNameForResourceShapeService(anAdaptorInterface) /];
 [for (aService: Service | anAdaptorInterface.serviceProviders().services) separator(lineSeparator())]import [javaClassFullName(aService) /];[/for]
 [for (aWebService: WebService | anAdaptorInterface.webServices) separator(lineSeparator())]import [javaClassFullName(aWebService) /];[/for]
 [if (anAdaptorInterface.trsServiceEnabled())]
-import org.glassfish.hk2.api.Factory;
```

Comment:
Are these unused when TRS is generated? I thought we generated some factories for TRS services?

Reply:
These are used for TRS. But the whole injection now occur in the Binder class.